packages/react-components/lib/robots/robot-info.tsx

```diff
@@ -15,6 +15,27 @@ function getTaskStatusDisplay(assignedTask?: string, taskStatus?: string) {
   }
 }
 
+function calculateLastUpdateTime(
```

Comment:
I tested it with the deployment setup, in realtime. this should be the equation. which just required one input arg of `updateTime`

```js
function calculateLastUpdateTime(
  updateTime: number,
): string {
    const lastUpdateMillis = Date.now() - updateTime;
    if (lastUpdateMillis < 1000) {
      return '< 1';
    } else {
      return (lastUpdateMillis / 1000.0).toFixed(1).toString();
    }
}
```